ajax/libs/country-region-dropdown-menu/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "country-region-dropdown-menu",
   "filename": "geodatasource-cr.min.js",
-  "version": "1.2.2",
+  "version": "2.1.1",
```

```suggestion
  "version": "1.2.2",
```
Please revert the change to the version number.